frontend/public/locales/en/public.json

```diff
@@ -1719,7 +1728,7 @@
   "Your 60-day self-support trial expired.": "Your 60-day self-support trial expired.",
   "Your 60-day self-support trial will end in < 1 day.": "Your 60-day self-support trial will end in < 1 day.",
   "Your 60-day self-support trial will end in {{count}} day on {{date}}.": "Your 60-day self-support trial will end in {{count}} day on {{date}}.",
-  "Your 60-day self-support trial will end in {{count}} day on {{date}}._plural": "Your 60-day self-support trial will end in {{count}} days on {{date}}.",
+  "Your 60-day self-support trial will end in {{count}} day on {{date}}._plural": "Your 60-day self-support trial will end in {{count}} day on {{date}}.s",
```

Comment:
This change surprises me, can you check why this value is changed?

Reply:
Fixed it.